packages/toolpad-app/src/toolpad/AppEditor/PageEditor/QueryEditor/QueryEditorDialog.tsx

```diff
@@ -106,6 +106,9 @@ export default function QueryNodeEditorDialog<Q>({
   const node = useLatest(nodeProp);
 
   const [input, setInput] = React.useState<appDom.QueryNode<Q>>(node);
+  React.useEffect(() => {
```

Comment:
This is repeated, there's the same thing in line 237 (I might've added in the last PR I merged).

Reply:
nvm different component :D

Reply:
Ok, I'm removing the other one to keep it closer to where the state is created.